Jenkinsfile.release

```diff
@@ -123,17 +121,16 @@ void pushImage(String imageName) {
                    docker trust key load $NEXUS_IQ_SERVER_REPOSITORY_KEY
                    docker trust key load $SONATYPE_KEY"""
 
-      // add signer - for this you need signers public key and repository keys password
       withCredentials([string(credentialsId: 'nexus-iq-server_dct_reg_pw', variable: 'DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE')]) {
-        runSafely "docker trust signer add sonatype ${imageName} --key $SONATYPE_PUB"
-      }
-
-      runSafely "docker tag ${env.DOCKER_IMAGE_ID} ${imageName}:${env.VERSION}"
-      runSafely "docker tag ${env.DOCKER_IMAGE_ID} ${imageName}:latest"
-
-      withCredentials([string(credentialsId: 'sonatype-password', variable: 'DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE')]) {
-        runSafely "docker image push ${imageName}:${env.VERSION}"
-        runSafely "docker image push ${imageName}:latest"
+        sh  """
+        curl -L https://go.dev/dl/go1.23.3.linux-amd64.tar.gz | tar -xzf -
+        export PATH=${env.PATH}:${env.WORKSPACE}/go/bin:${env.WORKSPACE}/bin
+        export GOPATH=${env.WORKSPACE}
+        go install -tags pkcs11 github.com/theupdateframework/notary/cmd/notary@latest
+        notary --help
```

Comment:
`notary --help` - Is this line needed here?

Reply:
Not strictly needed. I liked it because it helped me debug locally to know it was installed and setup correctly.